readme.md

```diff
@@ -0,0 +1,51 @@
+# PopojiCMS
+Free Content Management System - Indonesia
+Contact : info@popojicms.org
+
+## Cara instalasi PopojiCMS
+
```

Comment:
Oh yah, I forgot to include other option (download and clone from github). The download mentioned means download button in PopojiCMS official website.